crates/cheatable-starknet/src/lib.rs

```diff
@@ -20,8 +23,17 @@ impl CheatedState {
             rolled_contracts: HashMap::new(),
             pranked_contracts: HashMap::new(),
             warped_contracts: HashMap::new(),
+            deploy_counter: 0,
         }
     }
+
+    pub fn increment_deploy_counter(&mut self) -> () {
+        self.deploy_counter += 1;
+    }
+
+    pub fn gen_salt(&self) -> ContractAddressSalt {
+        ContractAddressSalt(StarkFelt::from(333_u32 + self.deploy_counter))
```

Comment:
Let me know is it ok? @piotmag769 @Arcticae

Reply:
Seems fine

Reply:
Why do we use `333`? :hmmm:

Reply:
Can't this be just `deploy_counter` ?

Reply:
good idea-> it's less confusing!